eth-supply/substreams.yaml

```diff
@@ -0,0 +1,42 @@
+specVersion: v0.1.0
+package:
+  name: eth_supply
+  version: v0.1.0
+
+imports:
+  eth: "https://github.com/streamingfast/sf-ethereum/releases/download/v0.10.2/ethereum-v0.10.4.spkg"
+
+binaries:
+  default:
+    type: wasm/rust-v1
+    file: "../target/wasm32-unknown-unknown/release/eth_supply.wasm"
+
+modules:
+  - name: map_supply_delta
+    kind: map
+    initialBlock: 0
+    inputs:
+      - source: sf.ethereum.type.v2.Block
+    output:
+      type: "proto:messari.eth_supply.v1.EthSupply"
+
+  - name: store_supply
+    kind: store
+    updatePolicy: add
+    valueType: bigint
+    inputs:
+      - map: map_supply_delta
+
+  - name: map_output
+    kind: map
+    initialBlock: 0
```

Comment:
0 is the default here. Only want to specify when wanting non zero starting blocks